exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/script_languages_container/customize.ipynb

```diff
@@ -0,0 +1,300 @@
+   "outputs": [],
+   "source": [
+    "%run ./utils/file_system_ui.ipynb\n",
+    "%run ./utils/slc_ui.ipynb"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "b632c4b5-3bb9-46c1-8c68-009102432a71",
+   "metadata": {},
+   "source": [
+    "## Customize\n",
+    "\n",
+    "First you need to define an alias for the new script-languages-container. The alias will be used to reference the container later from the UDF's.\n",
+    "\n",
+    "Note: In this tutorial the alias also will be used as part of the export file (tar.gz) and the uploaded container to the BucketFS. This allows you to create, upload and use different script-language-containers: one per alias."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "a88f47d9-7337-45fb-b2d4-08b74bc95f5f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "display(get_alias_ui(ai_lab_config, \"ai_lab_default_custom\"))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "410312a2-0976-423a-bdf5-2100a0085528",
+   "metadata": {},
+   "source": [
+    "### Flavor Customization Build Step\n",
+    "\n",
+    "`exasclt` consists of multiple build steps. By a build step here we mean a file structure which serves as an input for a certain stage of the building process of the script-languages-container. See [Advanced Topics](./advanced.ipynb) for more details.\n",
+    "\n",
+    "The **flavor_customization** build step consists of a Dockerfile and several package lists which can be modified. We recommend to use the package lists to add new packages to the flavor and only modify the Dockerfile if you need very specific changes, like adding additional resources."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ed0ad229-06b6-4083-8daa-a23e176b59d4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "show_directory_content(slctmanager.slc_dir.flavor_dir / \"flavor_customization\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8b0f6884-8fda-44d1-92a4-ae23040b109c",
+   "metadata": {},
+   "source": [
+    "The Dockerfile consists of two parts. The first part installs the packages from the package list and should only be changed with care. The second part is free for your changes. Read the description in the Dockerfile carefully to find out what you can and shouldn't do."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65cc2597-cb90-494c-af6e-e8a7c51f4130",
+   "metadata": {},
+   "source": [
+    "#### Package Lists\n",
+    "The package lists have a unified format. Each line consists of the package name and the package version separated by \"|\", e.g `xgboost|1.3.3`. You can comment out a whole line by adding\"#\" at the beginning. You can also add a trailing comment to a package definition by adding a \"#\" after the package definition. We usually recommend to install a specific package version, such that there are no surprises for which version gets installed."
```

Comment:
```suggestion
    "The package lists have a unified format. Each line consists of the package name and the package version separated by the pipe character `|`, e.g `xgboost|1.3.3`. You can comment out a whole line by adding a hash character `#` the beginning of the line. You can also add a trailing comment to a package definition by adding `#` after the package definition. We usually recommend to install a specific package version to avoid surprises about which version actually gets installed."
```

Reply:
fixed